src/config/openapi.yml

```diff
@@ -0,0 +1,71 @@
+openapi: 3.0.0
+info:
+  title: uspace API docs
+  version: 1.0.0
+paths:
+  "/s3":
+    get:
```

Comment:
This is still old yml 🤔 its alright, but lets make sure that FE and BE have the same thing.